alphastats/utils.py

```diff
@@ -73,34 +73,35 @@ def inner(*args, **kwargs):
     return inner
 
 
+# TODO: unused
 def list_to_tuple(function):
     """
     list are not hashable not suitable for caching
     convert to tuple
     """
 
     def wrapper(*args):
-        args = [tuple(x) if type(x) == list else x for x in args]
+        args = [tuple(x) if isinstance(x, list) else x for x in args]
         result = function(*args)
-        result = tuple(result) if type(result) == list else result
+        result = tuple(result) if isinstance(result, list) else result
         return result
 
     return wrapper
 
 
-def find_duplicates_in_list(l: list) -> list:
-    """Find duplicates in a list
+def find_duplicates_in_list(input_list: list) -> list:
```

Comment:
This function is only used once to assert index uniqueness, and should be replaced with https://pandas.pydata.org/docs/reference/api/pandas.Index.has_duplicates.html#pandas.Index.has_duplicates